install/docker/debian-docker-install.md

```diff
@@ -0,0 +1,121 @@
+# Install Docker for Kata Containers on Debian
+
+> **Note:**
+>
+> - This guide assumes you have
+>   [already installed the Kata Containers packages](../debian-installation-guide.md).
+> - this guide allows for installation with `systemd` or `sysVinit`init systems 
+> - notes are also provided for running multiple runtimes
+
+1. Install the latest version of Docker with the following commands:
+
+   > **Note:** This step is only required if Docker is not installed on the system.
+
+   ```bash
+   $ sudo apt-get install apt-transport-https ca-certificates curl gnupg2 software-properties-common  
+   $ curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg | sudo apt-key add -
+   $ sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") $(lsb_release -cs) stable"
+   $ sudo apt-get update
+   $ sudo apt-get install docker-ce
+   ```
+
+   For more information on installing Docker please refer to the
+   [Docker Guide](https://docs.docker.com/engine/installation/linux/debian).
+
+2. Configure Docker to use Kata Containers by default with one of the following methods:
+
+    > **Note:** It is possible to have two docker runtimes enabled and pick the option at config/start of a docker machine.
+    > - This is useful for running Clear Containers and Kata Containers concurrently, for instance.
+    > - see [Docker Docs on runtime options](https://docs.docker.com/engine/reference/commandline/dockerd/#docker-runtime-execution-options)
+
+    1. sysVinit
+    
+        - with sysVinit,  docker config is stored in `/etc/default/docker`, edit the options similar to the following: 
+        
+        ```
+        # single runtime
+        DOCKER_OPTS="-D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime
+        # multiple runtime options (kata default)
+        #DOCKER_OPTS="-D --add-runtime kata-runtime=/usr/bin/kata-runtime cc-runtime=/usr/bin/cc-runtime --default-runtime=kata-runtime
```

Comment:
I'd simplify this section by just specifying the absolute essentials. I think configuring multiple runtimes should be outside the scope of this docuemtn (and if users wish to do this, they can refer to the official Docker docs).

Further, the [Clear Containers project](https://github.com/clearcontainers) is no longer being maintained, so we do not wish to suggest it should be being used any more.

Reply:
OK